k8s/admin/monitoring/patches/cftptl/cluster-00/prom-operator.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: prom-operator
+  namespace: monitoring
+  annotations:
+    flux.weave.works/automated: "false"
+    flux.weave.works/ignore: "false"
+spec:
+  values:
+    prometheus:
+      ingress: 
+        enabled: true
+        hosts: 
+          - prometheus-ptl.sandbox.platform.hmcts.net
+
+    alertmanager:
+      ingress:
+        enabled: true
+        hosts:
+          - alertmanager-ptl.sandbox.platform.hmcts.net
+
+    grafana:
+      plugins:
+        - grafana-kubernetes-app
+        - camptocamp-prometheus-alertmanager-datasource
+      ingress:
+        enabled: true
+        annotations:
+          kubernetes.io/ingress.class: traefik
+        hosts:
+          - grafana-ptl.platform.hmcts.net
+      additionalDataSources: []
+
+      dashboardProviders: 
```

Comment:
Don't see anything specific to env in here? can the sections below be moved to base ?

Reply:
additionalDataSources is environment specific and it is only used by the MGMT clusters that host Grafana. Grafana will then use the datasource to access the prometheus instance on the cluster (i.e. MGMT SBOX needs sandbox01, sandbox00 as data sources)